RELEASE_NOTES.md

```diff
@@ -4,6 +4,22 @@ This page describes the noteworthy improvements provided by each release of Ecli
 
 ## 3.0.0 (under development)
 
+### New option to transform P2 dependencies into real maven coordinates
+
+The `tycho-consumer-pom` mojo has a new option to resolve p2 introduced dependencies to 'real' maven coordinates now, when enabled it queries maven-central with the SHA1 of the file to find out what are the actual maven central coordinates
+ and place them in the generated pom consumer-pom.
+
+```
+<plugin>
+	<groupId>org.eclipse.tycho</groupId>
+	<artifactId>tycho-packaging-plugin</artifactId>
+	<version>${tycho-version}</version>
+	<configuration>
+		<resolveP2Dependencies>true</resolveP2Dependencies>
```

Comment:
IMO, this should be `true` by default.

Reply:
If it is great and everyone loves it maybe... but I better have it disabled by default to not get complains, e.g because this do not currently uses any configured mirrors, proxies, authentication, offline mode and so on ...

And it only works with access to maven-central at the moment...

Reply:
You're too shy ;)
I think the name isn't explicit about what it does. Basically `resolve` is not something we can use in Tycho and hope it's clear enough, as Tycho resolves tons of things in tons of contexts. Maybe something like `preferRemoteArtifactGAV` ?

Reply:
actually it is some kind of reverse-resolve but this also sounds strange :-)

Reply:
I think I will name this `mapP2Dependencies` that seems a better name here...